symbolic-symcache/src/new/writer.rs

```diff
@@ -67,72 +81,30 @@ impl SymCacheConverter {
     /// If the string was already present, it is not added again. A newly added string
     /// is prefixed by its length as a `u32`. The returned `u32`
     /// is the offset into the `string_bytes` field where the string is saved.
-    fn insert_string(&mut self, s: &str) -> u32 {
+    fn insert_string(
+        string_bytes: &mut Vec<u8>,
+        strings: &mut HashMap<String, u32>,
+        s: &str,
+    ) -> u32 {
```

Comment:
I assume this is some sort of borrow checker dance?

Reply:
yup, since I can’t borrow all of `self` :-(